client/src/components/WorkflowInvocationState/WorkflowInvocationHeader.vue

```diff
@@ -23,6 +31,31 @@ const props = defineProps<Props>();
 
 const { workflow } = useWorkflowInstance(props.invocation.workflow_id);
 
+const userStore = useUserStore();
+const owned = computed(() => {
+    if (userStore.currentUser && workflow.value) {
+        return userStore.currentUser.username === workflow.value.owner;
+    } else {
+        return false;
+    }
+});
+
+async function onImport() {
+    if (!workflow.value || !workflow.value.owner) {
+        return;
+    }
+    try {
+        await copyWorkflow(workflow.value.id, workflow.value.owner);
+        Toast.success(
```

Comment:
Can we do something else here ? The toast is so far away from where you clicked, especially since it contains a link to be clicked. Maybe use b-alert with info styling ?

Reply:
Sure we could use a `BAlert`, but what about instead, just rerouting as soon as an import is successful?

Reply:
**Update:** Replaced the `Toast` with an alert (see updated GIF above)